Fixed #33764 deprecate make_random_password (#15752)
django/django: docs/releases/4.2.txt

Miscellaneous
-------------

> * ``User.objects.make_random_password()`` generates a password that doesn't
>   comply with most password validators.

Reviewer: Maybe something like
```suggestion
* The ``make_random_password`` method of ``User.objects`` is deprecated.
```
